components/autofill/src/sync/engine.rs

```diff
@@ -81,6 +81,7 @@ impl<T> ConfigSyncEngine<T> {
         let tx = db.unchecked_transaction()?;
         self.storage_impl.reset_storage(&tx)?;
         self.put_meta(&tx, LAST_SYNC_META_KEY, &0)?;
+        tx.commit()?;
```

Comment:
I think this might have been my first contribution to app-services :joy:

Reply:
Welp, this component was my first time implementing standalone effort so even? 😬